linkerd/app/inbound/src/metrics/authz.rs

```diff
@@ -0,0 +1,153 @@
+use crate::policy::{AllowPolicy, Permit};
+use linkerd_app_core::{
+    metrics::{metrics, AuthzLabels, Counter, FmtMetrics, ServerLabel},
+    transport::labels::TargetAddr,
+};
+use parking_lot::RwLock;
+use std::{collections::HashMap, sync::Arc};
+
+metrics! {
+    inbound_http_authz_allow_total: Counter {
+        "The total number of inbound HTTP requests that were authorized"
+    },
+    inbound_http_authz_deny_total: Counter {
+        "The total number of inbound HTTP requests that could not be processed due to a proxy error."
+    },
+
+    inbound_tcp_authz_allow_total: Counter {
+        "The total number of inbound HTTP requests that were authorized"
+    },
+    inbound_tcp_authz_deny_total: Counter {
+        "The total number of inbound HTTP requests that were denied"
+    },
+    inbound_tcp_authz_terminate_total: Counter {
+        "The total number of inbound HTTP requests that were terminated due to an authorization change"
+    }
```

Comment:
looks like the help messages were copied and pasted incorrectly:
```suggestion
    inbound_tcp_authz_allow_total: Counter {
        "The total number of inbound TCP connections that were authorized"
    },
    inbound_tcp_authz_deny_total: Counter {
        "The total number of inbound TCP connections that were denied"
    },
    inbound_tcp_authz_terminate_total: Counter {
        "The total number of inbound TCP connections that were terminated due to an authorization change"
    }
```